src/core/tests/widgets/test_canvas.py

```diff
@@ -432,19 +432,20 @@ def test_fill_repr(self):
 
     def test_stroke_modify(self):
         with self.testing_canvas.stroke(
-            color=BLANCHEDALMOND, line_width=5.0
+            color=BLANCHEDALMOND, line_width=5.0, line_dash=[2,2]
         ) as stroker:
             stroker.color = REBECCAPURPLE
             stroker.line_width = 1
+            stroker.line_dash = None
             self.testing_canvas.redraw()
         self.assertActionPerformedWith(
-            self.testing_canvas, "stroke", color=rgb(102, 51, 153), line_width=1
+            self.testing_canvas, "stroke", color=rgb(102, 51, 153), line_width=1, line_dash=None
```

Comment:
This seems counterintuitive - shouldn't the stroke action be performed with the dash that was provided?